### DISCUSSION TITLE
Modify time of backup/optimized

### DISCUSSION THREAD

**Discussion by nicolasforest** [sethgoldin/davinci-resolve-postgresql-workflow-tools#39] [General]
Hi, me again. Is there a way to change the time where the backup and the optimized will happen?
I've setup the backup yesterday around 10:30AM, but today, everybody get kicked out of their session when the backup/optimized happen at 10:30.

If I want to change the time at 3AM in the morning, any way to do it? I've took a look in the .plist in LaunchDeamon but there is no time associate in the document.

**Comment by sethgoldin** [comment]
This doesn't make any sense. Sounds like something else is going on. [`pg_dump`](https://www.postgresql.org/docs/13/app-pgdump.html) should work even if users are connected. It shouldn't be kicking users out of their sessions. Sounds like something else is going on, and you should figure out whatever the root issue is.

By default, the launchd daemons for macOS are using the `StartInterval` key, but you could go [modify it manually](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html) however you like.
